File: Projeto_WindowsForms_Web/Modelo/Colaborador.cs
using Modelo.Enum;

namespace Modelo
{
    public class Colaborador : ModeloBase
    {
        public string NomeCompleto { get; set; }
        public TipoSexo Sexo { get; set; }
        public TipoCargo Cargo { get; set; }
        public decimal Salario { get; set; }
        public DateTime DataAdmissao { get; set; }
        public Empresa Empresa { get; set; }
    }
}

File: Projeto_Android_Studio/MyApplication/app/src/main/java/com/example/myapplication/modelo/Colaborador.java
package com.example.myapplication.modelo;

import com.example.myapplication.modelo.enums.TipoCargo;
import com.example.myapplication.modelo.enums.TipoSexo;
import java.util.Date;

public class Colaborador extends BaseModel {
    public String NomeCompleto;
    public TipoSexo Sexo;
    public TipoCargo Cargo;
    public Double Salario;
    public String DataAdmissao;
    public Empresa Empresa;
}
